gapic-generator/lib/gapic/presenters/method_rest_presenter.rb

```diff
@@ -27,14 +27,17 @@ class MethodRestPresenter
 
       attr_reader :http_bindings
 
+      # @return [String] String representation of this presenter type.
+      attr_reader :type
+
       ##
       # @param main_method [Gapic::Presenters::MethodPresenter] the main presenter for this method.
       # @param api [Gapic::Schema::Api]
-      #
```

Comment:
Just as a formatting note, we do intentionally have this across most of our code base. The idea is to put a bit of vertical space between the documentation and the method (or class or anything else) being documented, to break up the "wall of text" a bit. So most of our methods look something like this:

```ruby
##
# This is a description of a method
#
# @param name [String] This is what a param looks like
# @return [Object] This might be a return value
#
def do_something name
  foo
  bar
end
```

The only usual exception is occasionally for a single-line thing like an `attr_reader` (e.g. what you have a few lines further up) where the documentation is also single-line, we might dispense with both the blank comment line above and below to save space.

Reply:
Updated some docstrings for consistency. Thanks!